### PR TITLE
Adding a gauge for seconds since last merge.

### PR DESCRIPTION
**Pull request by Q-Lee**
It would be nice to alarm on time since last merge.

### CODE REVIEW

File: mungegithub/mungers/submit-queue.go [hunk 1]
```diff
@@ -552,6 +558,7 @@ func (sq *SubmitQueue) EachLoop() error {
 	sq.updateHealth()
 	promMetrics.OpenPRs.Set(float64(len(sq.prStatus)))
 	promMetrics.QueuedPRs.Set(float64(len(sq.githubE2EQueue)))
+	promMetrics.MergeDelay.Set(float64(time.Now().Sub(sq.lastMergeTime).Seconds()))
```
Comment (rmmh) [inline mungegithub/mungers/submit-queue.go]: put this in the updateMergeRate function, after `promMetrics.MergeCount.Inc()`

**Comment by fejta** [comment]
Is this still alive? Please update and/or close

**Pull request closed without merging**